### PR TITLE
Remainder algorithm for Integer

### PR DESCRIPTION
**Pull request by mirec12590**
I created remainder algorithm for Integer values

### CODE REVIEW

File: src/Core/Echo.Concrete/Values/ValueType/IntegerValue.cs [hunk 1]
```diff
@@ -622,6 +622,73 @@ public virtual void Divide(IntegerValue other)
...
+                {
+                    if (firstNum.GetBit(i) == Trilean.Unknown)
+                        firstNum.SetBit(i, Trilean.True);
+
+                    if (secondNum.GetBit(i) == Trilean.Unknown)
+                        secondNum.SetBit(i, Trilean.True);
+                }
+
+                // Result is shorter number
+                result = (firstNum.IsGreaterThan(secondNum, false)) ? (IntegerValue) secondNum.Copy() : (IntegerValue) firstNum.Copy();
```
Comment (zsr2531) [inline src/Core/Echo.Concrete/Values/ValueType/IntegerValue.cs:669]: - Ternary operator style should be like this for better readability:
	```
	condition
		? case1
		: case2;
	```
- Redundant parentheses around condition

**Review by zsr2531 — APPROVED** [review]
One minor little style issue, apart from that looks :ok_hand:! Nicely done!

**Comment by mirec12590** [comment]
It is done 😄

**Comment by zsr2531** [comment]
Thanks!

**Comment by zsr2531** [comment]
Closes #19.

**Pull request merged**